01-JS-DOM-APIs/Exercise6/js/main.js

```diff
@@ -0,0 +1,38 @@
+window.onload = function(){
+    document.getElementById('btn').addEventListener("click", tableGenerator);
+
+    function tableGenerator(){
+    
+    let clean = document.getElementById('table');
+    clean.innerHTML = '';
+
+    var body = document.getElementsByTagName("body")[0];
+    var table = document.getElementById("table");
+    tables = document.createElement("table");
+    bodyTable = document.createElement("tbody");
+    
+    let matriz = [2];
+    matriz[0] = document.getElementById('rows').value;
+    matriz[1] = document.getElementById('columns').value;
+
+    for (var i = 0; i < matriz[0]; i++) {
+        tr = document.createElement("tr");
+        for (var j = 0; j < matriz[1]; j++) {
+
+                td = document.createElement("td");
+                textTd = document.createTextNode("Row: "+i+". Column: "+j);
+
+                td.appendChild(textTd);
+                tr.appendChild(td);
+
+        }
+
+            bodyTable.appendChild(tr);
+    }
+
+    table.appendChild(bodyTable);
+    table.appendChild(tables);
```

Comment:
Tables is not being manipulated. Check the DOM result

![image](https://user-images.githubusercontent.com/18276548/40145504-7f822c74-5938-11e8-9b20-19bd385c1b76.png)